src/components/implementation/no_interface/cFE_booter/osfiles.c

```diff
@@ -270,16 +330,23 @@ int32 OS_rmfs(char *devname)
 int32 OS_unmount(const char *mountpoint)
 {
     if (!mountpoint) return OS_FS_ERR_INVALID_POINTER;
-    return unmount(mountpoint);
+    int32 ret = path_chk_exists(mountpoint);
+    if (ret != OS_FS_SUCCESS) return ret;
+    return fs_unmount((char *)mountpoint);
 }
 
 /*
  * Returns the number of free blocks in a file system
 */
 int32 OS_fsBlocksFree(const char *name)
 {
-    PANIC("Unimplemented method!"); // TODO: Implement me!
-    return 0;
+   /*
+    * I am open to discussion on how we want to handle this.
+    * We could impose an artificial limit on the size of the filesystem
+    * if we expect the filesystem to be used much, that is the right choice, but
+    * otherwise it is not worth actually doing
+    */
+    return OS_ERR_NOT_IMPLEMENTED;
```

Comment:
So do you guys think imposing an artificial limit on the amount of space being used is the right choice.  And then just count the number of bump page allocations?